app/routes/results.py

```diff
@@ -13,13 +13,18 @@
 def getresults(id=0):
     try:
         user = get_current_user()
-        project = get_project_id(request.args.get("project"))
+        project = get_project_id(request.args.get("project")) 
+        page_no = request.args.get("page") or None
+        row_size = request.args.get("pageSize") or None
```

Comment:
I think, In row_size, instead of None directly add default size i.e. 20, so in ResultModel.py code may be reduced (line no. 62 to 68).

```suggestion
        row_size = request.args.get("pageSize") or 20
```